src/admin/content/views/ContentEdit.tsx

```diff
@@ -145,6 +145,43 @@ const ContentEdit: FunctionComponent<ContentEditProps> = ({ history, match, user
 		}
 	}, [id, user, t]);
 
+	const onPasteContentBlock = useCallback(
+		(e: any) => {
```

Comment:
Not 100% sure, maar ik herinner me het type `ClipboardEvent`. Seems like I forgot to use it.